packages/calling/src/Contacts/ContactsClient.ts

```diff
@@ -267,12 +255,7 @@ export class ContactsClient implements IContacts {
   }
 
   /**
-   * @param contact - Contact object.
-   * @param contactsDataMap - Object with contactId as key and contact info for corresponding contact as value.
-   * @param avatarUrlDomain - Avatar Url.
-   * @param encryptionKeyUrl - Encryption key received from KMS.
-   * @param ownerId - OwnerId.
-   * @returns Array of contact detail fetched from DSS.
+   * Fetches contacts from DSS.
```

Comment:
Question here. @mkesavan13 @Kesari3008 should we document private functions too? They're private right.. we don't want developers to really work with them. Moreover, dss is an unknown thing to an external dev?

Reply:
@sreenara - It's okay to document these but better to have them `@ignore`-d in the documentation.

I think I've left a bunch of similar comments on my previous review